ci/autoscaler/pipeline.yml

```diff
@@ -298,25 +292,26 @@ jobs:
   - in_parallel:
     - get: bbl-state
       passed: [deploy-cf]
-    - get: app-autoscaler-release-main
-      passed: [unit-tests-main, integration-tests-main]
```

Comment:
hmmm ... not sure about removing the unit tests ... Im guessing you want a separate resource to pin ?

Reply:
As the deploy_autoscaler job is only for testing purposes no real need to run the unit tests before hand, that can be saved for the test/release part of the pipeline.